### PR TITLE
Add tests for invalid and duplicate certs in the repo

### PR DESCRIPTION
**Pull request by jmhooper**
**Why**: To make sure we don't check in invalid or duplicate certs going forward.

This commit also adds a rake task to remove invalid certs

### CODE REVIEW

File: spec/certs/store_spec.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+describe 'Certificate store in config/certs' do
+  before do
+    # We need to allow net connect to download CRLs and check for revocations
+    WebMock.allow_net_connect!
```
Comment (zachmargolis) [inline spec/certs/store_spec.rb:6]: Would it be worth the effort to whitelist domains for this?

```
Webmock.disable_net_connect!(allow: [
  'ocsp.lol.gov',
  'certs.mil'
])
```

Comment (jmhooper) [reply spec/certs/store_spec.rb]: I thought there'd be a bunch but after digging in there are only 4. I added them to an allow list.

**Comment by jmhooper** [comment]
Weirdly, this is passing locally, but they're all failing in CI. Worse, they all work in CI when I run the build with SSH access 🤔

**Comment by jmhooper** [comment]
It's the test order. Seed `61549` reliably reproduces it.

File: spec/certs/store_spec.rb [hunk 2]
```diff
@@ -11,6 +11,10 @@
     )
 
     CertificateStore.instance.reset
+    CertificateStore.instance.clear_root_identifiers
```
Comment (zachmargolis) [inline spec/certs/store_spec.rb:14]: yesssss test pollution :[ .... wouldn't we expect `#reset` to do all these things?

**Comment by jmhooper** [comment]
Fixed. The problem was I needed to reset the certificate store and clear its root IDs.

I also reset the OCSP and revocation caches for good measure.

File: spec/certs/store_spec.rb (hunk 2 above)
Comment (jmhooper) [reply spec/certs/store_spec.rb:14]: I think clearing root identifiers make sense. Idk about busting the caches though.

Comment (jmhooper) [reply spec/certs/store_spec.rb:14]: Maybe we move these to a global before action?

Comment (jmhooper) [reply spec/certs/store_spec.rb:14]: Except reseting the CertStore since rebuilding it is slow.
